semantic-core/src/Analysis/Typecheck.hs

```diff
@@ -44,6 +45,9 @@ data Monotype f a
   | Record (Map.Map User (f a))
   deriving (Foldable, Functor, Generic1, Traversable)
 
+-- FIXME: Union the effects/annotations on the operands.
+deriving via (Last (Term Monotype a)) instance Semigroup (Term Monotype a)
```

Comment:
I was legit surprised that this instance isn’t orphaned, but it’s due to `Monotype` being defined in this module.